### PR TITLE
docs: updating the docs with the http secure tag

### PR DESCRIPTION
**Pull request by aanchal88**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/marketing/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The `contributors.json` should be maintained to keep our "org chart" in a single
 There are two pages:
 
 *   https://developers.google.com/experts/all/technology/angular
-(Googlers: source at http://google3/googledata/devsite/content/en/experts/all/technology/angular.html) which is maintained by Dawid Ostrowski based on a spreadsheet https://docs.google.com/spreadsheets/d/1_Ls2Kle7NxPBIG8f3OEVZ4gJZ8OCTtBxGYwMPb1TUVE/edit#gid=0.
+(Googlers: source at https://google3/googledata/devsite/content/en/experts/all/technology/angular.html) which is maintained by Dawid Ostrowski based on a spreadsheet https://docs.google.com/spreadsheets/d/1_Ls2Kle7NxPBIG8f3OEVZ4gJZ8OCTtBxGYwMPb1TUVE/edit#gid=0.
```
Comment (jessicajaniuk) [inline aio/content/marketing/README.md:12]: Actually, can you just remove this entire line? It's no longer useful.

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
This looks great and should be good. If you could remove that line, I'll approve it. Thanks for your work here!

File: aio/content/marketing/README.md (hunk 1 above)
Comment (aanchal88) [reply aio/content/marketing/README.md]: Hii @jessicajaniuk , updated the change suggested. Please review.

**Comment by aanchal88** [comment]
Hii @jessicajaniuk , updated the change suggested. Please review.

**Comment by aanchal88** [comment]
Hii @jessicajaniuk , I am closing this PR as by mistake multiple commits has been done.
Opening another PR https://github.com/angular/angular/pull/50696 for the same. Please have a look.
Thanks

**Comment by aanchal88** [comment]
Hii @jessicajaniuk , I am closing this PR as by mistake multiple commits have been done.
Opening another PR https://github.com/angular/angular/pull/50696 for the same. Please have a look.
Thanks

**Pull request closed without merging**